doc/api/util.md

```diff
@@ -286,13 +286,31 @@ invoke and use the result of when inspecting the object:
 ```js
 const util = require('util');
 
-const obj = { name: 'nate' };
-obj[util.inspect.custom] = function(depth) {
-  return `{${this.name}}`;
-};
+class Box {
+  constructor(value) {
+    this.value = value;
+  }
 
-util.inspect(obj);
-  // "{nate}"
+  inspect(depth, options) {
```

Comment:
@Havvy .. as a suggestion per @fhinkel's request, a code comment in here that describes the basic flow would be helpful.